app/src/app/[lng]/[inventory]/TabHeader.tsx

```diff
@@ -0,0 +1,81 @@
+import { TFunction } from "i18next";
+import { Badge, Box, Heading, HStack, Link, Text } from "@chakra-ui/react";
+import { Trans } from "react-i18next/TransWithoutContext";
+import { MdOutlineAccountTree, MdOutlineCalendarToday } from "react-icons/md";
+
+export function TabHeader({
+  t,
+  title,
+  year,
+}: {
+  t: TFunction<string, undefined>;
+  year: number | undefined;
+  title: string;
+}) {
+  return (
+    <>
+      <Box className="flex items-center gap-3">
+        <Heading fontSize="headline.sm" fontWeight="semibold" lineHeight="32">
+          <Trans t={t}>{title}</Trans>
```

Comment:
This is sufficient, since there seem to be no components in the output (for which you'd need the `Trans` component).
```suggestion
          {t(title)}
```